dist/docker/debian/build_docker.sh

```diff
@@ -76,7 +76,7 @@ run apt-get -y update
 run apt-get -y install dialog apt-utils
 run bash -ec "echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections"
 run bash -ec "rm -rf /etc/rsyslog.conf"
-run apt-get -y install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python2.7 python3 python3-yaml curl rsyslog locales sudo
+run apt-get -y install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python2 python3 python3-yaml curl rsyslog locales sudo
```

Comment:
I thought the idea was to remove python3.

Reply:
The python3 was removed from `scylla-tools-java` dependency. we need python2 for cqlsh, so i guess python3 can be removed

Reply:
The reason i didn't remove python3 is because of `python3-yaml` which is not available any more for python2. Although i am not sure we really need it

Reply:
So what's going on? You're removing python2.7. Why?

Reply:
Because `scylla-tools-java` has as a dependancy either `python` package (which is not available anymore) or `python2` package (see here: https://github.com/scylladb/scylla-tools-java/blob/b004da9d1bc80b5ced255a2de18b7e440dc7cf99/dist/debian/control.template#L20).

Reply:
Ok. It would be better to just let the dependencies work and pull the things they need.